tests/kernel/fifo/fifo_timeout/src/main.c

```diff
@@ -199,12 +199,14 @@ static int test_multiple_threads_get_data(struct timeout_order_data *test_data,
 
 		data = k_fifo_get(&timeout_order_fifo, K_FOREVER);
 
```

Comment:
Wouldn't it have been simpler just to put "if(!data) { return TC_FAIL; }" right here instead of mucking with the nested if structure?

Reply:
In this if(!data) { return TC_FAIL; } then TC_ERROR print should be deleted. Hence did not do this change. Thought that print should be necessary.

Reply:
Hi @andyross , I know you approved it, but this is not merged. Would like to know if I need to make changes you mentioned to merge this. I am new, trying to understand how merge happens usually?

Reply:
Anas generally picks these up and merges them once all the checks have passed and approval is done.  I'm not demanding changes here, it was just a nitpick.

Reply:
Cool thanks 👍 will wait.